packages/notifier/test/test-publish-kit.js

```diff
@@ -288,16 +283,17 @@ test('makePublishKit', async t => {
     undefined,
   };
   for (const [label, initialValue] of Object.entries(initialValues)) {
-    const { subscriber } = makePublishKit(initialValue);
+    const { publisher, subscriber } = makePublishKit();
+    publisher.publish(initialValue);
     // eslint-disable-next-line no-await-in-loop
     const subAll = await Promise.all([
       subscriber.subscribeAfter(),
-      subscriber.subscribeAfter(expectedInitialCount - 1n),
+      subscriber.subscribeAfter(0n),
     ]);
     const [subFirst] = subAll;
     t.like(subFirst, {
       head: { done: false },
-      publishCount: expectedInitialCount,
+      publishCount: 1n,
     });
```

Comment:
This test is assuming too much about the implementation details of `makePublishKit` rather than its external interface. We should just drop the `subscribeAfter` argument and `publishCount` assertion.
```suggestion
    const subAll = await Promise.all([
      subscriber.subscribeAfter(),
      subscriber.subscribeAfter(),
    ]);
    const [subFirst] = subAll;
    t.like(subFirst, {
      head: { done: false },
    });
```

Reply:
> This test is assuming too much about the implementation details

You mean the `publicCount: 1n` part? (The `0n` argument is legit.)

I agree the interface doesn't guarantee that the next value will be one greater. Because there is a `publish()` it does guarantee that it will be greater though, right?

When I read this test in master I thought it was about the counts after publish but the first test in the file checks those. This seems to really be testing publishing particular values (of the `initialValues`) loop. Since there is no "initial value" anymore, and I trust there are already tests for `.publish()`, I'll just remove this test.